CONTRIBUTING.adoc

```diff
@@ -207,6 +207,8 @@ use cases.
 To run a dind cluster in a VM, follow steps 1-3 of the Vagrant
 instructions and then execute the following:
 
+        # Extra memory is required to build the extended tests
+        $ export OPENSHIFT_MEMORY=3072
```

Comment:
With go1.5, this is needed in the non-dind deploy too. Maybe we should just bump the default.